accel/tcg/log_instr.c

```diff
@@ -642,8 +645,6 @@ static void do_cpu_loglevel_switch(CPUState *cpu, run_on_cpu_data data)
     qemu_log_instr_loglevel_t next_level = data.host_int;
     bool next_level_active;
 
-    log_assert(qemu_loglevel_mask(CPU_LOG_INSTR));
```

Comment:
Why was this removed? Is the assumption changed or was just redundant?

Reply:
The first I think? The assert was being hit on a normal path after the change.